spk/fuse-exfat/Makefile

```diff
@@ -0,0 +1,24 @@
+SPK_NAME = fuse-exfat
```

Comment:
In my mind, calling the SPK `exFAT` instead makes sense. Same with the filename, for that matter.